modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabaseImpl.java

```diff
@@ -84,6 +84,8 @@ public class SearchServiceDatabaseImpl implements SearchServiceDatabase {
   /** Logging utilities */
   private static final Logger logger = LoggerFactory.getLogger(SearchServiceDatabaseImpl.class);
 
+  private static final String ACL_ID_PREFIX_EPISODE = "ROLE_EPISODE_";
```

Comment:
Do you perhaps have a recommendations on where the constants/methods could live? I'd like to avoid creating unnecessary dependencies between asset manager and search if possible?

Reply:
Put them in common.